sdk/formrecognizer/Azure.AI.FormRecognizer/src/CustomFormClient.cs

```diff
@@ -1,369 +1,369 @@
-        /// </summary>
-        /// <param name="modelId">The ID of the model to delete.</param>
-        /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
-        /// <returns></returns>
-        public virtual async Task<Response> DeleteModelAsync(string modelId, CancellationToken cancellationToken = default)
-        {
-            return await _operations.DeleteCustomModelAsync(new Guid(modelId), cancellationToken).ConfigureAwait(false);
-        }
-
-        /// <summary>
-        /// Get a collection of <see cref="CustomModelInfo"/> items describing the models trained on this subscription
-        /// and their training status.
-        /// </summary>
-        /// <param name="cancellationToken"></param>
-        /// <returns></returns>
-        public virtual Pageable<CustomModelInfo> GetModelInfos(CancellationToken cancellationToken = default)
-        {
-            return _operations.GetCustomModelsPageableModelInfo(GetModelOptions.Full, cancellationToken);
-        }
-
-        /// <summary>
-        /// Get a collection of <see cref="CustomModelInfo"/> items describing the models trained on this subscription
-        /// and their training status.
-        /// </summary>
-        /// <param name="cancellationToken"></param>
-        /// <returns></returns>
-        public virtual AsyncPageable<CustomModelInfo> GetModelInfosAsync(CancellationToken cancellationToken = default)
-        {
-            return _operations.GetCustomModelsPageableModelInfoAsync(GetModelOptions.Full, cancellationToken);
-        }
-
-        /// <summary>
-        /// Get the number of models trained on this subscription and the subscription limits.
-        /// </summary>
-        /// <param name="cancellationToken"></param>
-        /// <returns></returns>
-        public virtual Response<SubscriptionProperties> GetSubscriptionProperties(CancellationToken cancellationToken = default)
-        {
-            Response<Models_internal> response = _operations.RestClient.GetCustomModels(GetModelOptions.Summary, cancellationToken);
-            return Response.FromValue(new SubscriptionProperties(response.Value.Summary), response.GetRawResponse());
-        }
-
-        /// <summary>
-        /// Get the number of models trained on this subscription and the subscription limits.
-        /// </summary>
-        /// <param name="cancellationToken"></param>
-        /// <returns></returns>
-        public virtual async Task<Response<SubscriptionProperties>> GetSubscriptionPropertiesAsync(CancellationToken cancellationToken = default)
-        {
-            Response<Models_internal> response = await _operations.RestClient.GetCustomModelsAsync(GetModelOptions.Summary, cancellationToken).ConfigureAwait(false);
-            return Response.FromValue(new SubscriptionProperties(response.Value.Summary), response.GetRawResponse());
-        }
-
-        #endregion CRUD Ops
-    }
-}
+//using System;
```

Comment:
Would it make sense to delete this file completely?

Reply:
I want to keep it around until I finish the rest of the training client as I keep going back and forth looking at it.
Will it be ok to leave it for this PR?

Reply:
Yep, np.